plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetastoreClosure.java

```diff
@@ -202,6 +203,9 @@ public void dropColumn(HiveIdentity identity, String databaseName, String tableN
             List<String> columnNames,
             TupleDomain<String> partitionKeysFilter)
     {
+        if (isPartitionKeyFilterFalse(partitionKeysFilter)) {
```

Comment:
I don't think it's needed here. The only caller that does not set `all` (`SemiTransactionalHiveMetastore#doGetPartitionNames`) already handles that case.

Reply:
Good point, removed